CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 [Full Changelog](In progress)
 
+### Logins
+New trait `NSSKeyManager`, which provides an NSS-backed key manager. Given a
```

Comment:
```suggestion
New trait `KeyManager`, which provides an NSS-backed key manager. Given a
```

Reply:
oh good catch, `NSSKeyManager` is not a trait, but an actual implementation of the `KeyManager` trait. Fixed this